src/polyfills/keyboardevent-key.js

```diff
@@ -44,9 +44,13 @@ const keys = {
  * @param {!Window} win
  */
 export function install(win) {
+  const isIe = /Trident|MSIE|IEMobile/i.test(win.navigator.userAgent);
```

Comment:
Can we leverage the service? 

https://github.com/ampproject/amphtml/blob/master/src/service/platform-impl.js#L93

Reply:
The service isn't available when the polyfills are loaded. I discussed with @aghassemi https://github.com/ampproject/amphtml/pull/18879#discussion_r227567395